qemu/tests/rng_hotplug.py

```diff
@@ -79,17 +83,19 @@ def stop_rngd(vm):
     login_timeout = int(params.get("login_timeout", 360))
     repeat_times = int(params.get("repeat_times", 1))
     rng_num = int(params.get("rng_num", 1))
-    test_before_hotplug = params.get("test_before_hotplug")
+    test_before_hotunplug = params.get("test_before_hotunplug")
     test_after_hotplug = params.get("test_after_hotplug")
```

Comment:
test_after_hotplug and sub_type_after_plug make the user confused

Reply:
I will keep the original parameter test_before_hotunplug;
and rename sub_type_after_plug to pm_test_after_plug,is this acceptable?

Reply:
What's pm_test?

Reply:
reboot/shutdownpower management test